gocd/gocd_test.go

```diff
@@ -86,6 +86,34 @@ func TestGocdGET(t *testing.T) {
 	}
 }
 
+// //Test require a pipeline to have a history
```

Comment:
Add in comment here briefly explaining why these are commented and what we could do to test in the future.